tests/exploreServicesBreakDown.spec.ts

```diff
@@ -1,7 +1,5 @@
-import pluginJson from '../src/plugin.json';
-import { test, expect } from '@grafana/plugin-e2e';
-import { ExplorePage } from './fixtures/explore';
-import { ROUTES } from '../src/services/routing';
+import {expect, test} from '@grafana/plugin-e2e';
```

Comment:
Oh weird, prettier doesn't seem to run on the e2e files

Reply:
I tried adding it to the `.lintstagedrc.js` but it wasn't working, maybe because it's outside the rootDir defined in the base .tsconfig? I manually linted for now, something to add to the todo list :/